docs/src/topics/export.md

```diff
@@ -10,98 +10,50 @@ u = rand(ndofs(dh)); σ = rand(getncells(grid))
-vtk_cell_data(vtk,  σ, "my-cell-data")
-# hide
-```
-
-Finally, we need to save the file to disk, using `vtk_save`
-
-```@example export
-vtk_save(vtk)
-rm("my-solution.vtu") # hide
-```
-
-Alternatively, all of the above can be done using a `do` block:
-
-```@example export
-vtk_grid("my-solution", grid) do vtk
-    vtk_point_data(vtk, u, "my-point-data")
-    vtk_cell_data(vtk, σ, "my-cell-data")
-end
-rm("my-solution.vtu") # hide
-```
-
-For other functionality, and more information refer to the
-[`WriteVTK.jl` README](https://github.com/jipolanco/WriteVTK.jl/blob/master/README.md).
-In particular, for exporting the solution at multiple time steps, the
-[section on PVD files](https://github.com/jipolanco/WriteVTK.jl#paraview-data-pvd-file-format)
-is useful.
-
-## Exporting with `DofHandler`
-
-There is an even more convenient way to export a solution vector `u` -- using the
-`DofHandler`. The `DofHandler` already contains all of the information needed,
-such as the names of our fields and if they are scalar or vector fields. But most
-importantly the `DofHandler` knows about the numbering and distribution of
-degrees of freedom, and thus knows how to "distribute" the solution vector on
-the grid. For example, lets say we have a `DofHandler` `dh` and a solution
-vector `u`:
-
+The following structure can be used to write various output to a vtk-file:
 ```@example export
-vtk = vtk_grid("my-solution", dh)
-vtk_point_data(vtk, dh, u)
-vtk_save(vtk)
-rm("my-solution.vtu") # hide
-```
-
-or with a `do`-block:
-
+VTKFile("my_solution", grid) do vtk
+    write_solution(vtk, dh, u)
+end;
+```
+where `write_solution` is just one example of the following functions that can be used 
+
+* [`write_solution`](@ref)
+* [`write_celldata`](@ref)
+* [`Ferrite.write_nodedata`](@ref)
+* [`write_projection`](@ref)
+* [`Ferrite.write_cellset`](@ref)
+* [`Ferrite.write_nodeset`](@ref)
+* [`Ferrite.write_dirichlet`](@ref)
+* [`Ferrite.write_cell_colors`](@ref)
+
+Instead of using the `do`-block, it is also possible to do
 ```@example export
-vtk_grid("my-solution", dh) do vtk
-    vtk_point_data(vtk, dh, u)
-    vtk_cell_data(vtk, σ, "my-cell-data")
+vtk = VTKFile("my_solution", grid)
+write_solution(vtk, dh, u)
+# etc.
+close(vtk);
+```
+
+The data written by `write_solution`, `write_celldata`, `Ferrite.write_nodedata`, and `write_projection` may be either scalar (`Vector{<:Number}`) or tensor (`Vector{<:AbstractTensor}`) data. 
+
+For simulations with multiple time steps, typically one `VTK` (`.vtu`) file is written 
+for each time step. In order to connect the actual time with each of these files,
+a `VTKFileCollection` can be used, which will write one paraview datafile (`.pvd`)
+file and one `VTKFile` (`.vtu`) for each time step. 
+
+```@example pvdexport 
+pvd = VTKFileCollection("my_results", grid)
+for t in range(0, 1, 5)
+    # Do calculations to update u
+    addstep!(pvd, t) do vtk
+        write_solution(vtk, dh, u)
+    end
```

Comment:
Perhaps this could be
```suggestion
    VTKFile(pvd, t) do vtk
        write_solution(vtk, dh, u)
    end
```
?

Reply:
The motivation for the `addstep!` function, or any other generic name, is that it allows replacing `pvd` with any other collection type, allowing you to only change the output file type in a single location.